pom.xml

```diff
@@ -89,9 +89,9 @@
     <version.build.helper.plugin>1.10</version.build.helper.plugin>
 
     <!-- Overridden from parent -->
-    <jdk.min.version>1.5</jdk.min.version>
-    <maven.compiler.target>1.5</maven.compiler.target>
-    <maven.compiler.source>1.5</maven.compiler.source>
+    <jdk.min.version>1.6</jdk.min.version>
+    <maven.compiler.target>1.6</maven.compiler.target>
+    <maven.compiler.source>1.6</maven.compiler.source>
```

Comment:
I'm not sure about bumping it to `1.6` as we somehow should also support `1.5` as far as I remember. But maybe @aslakknutsen can share his thoughts on this topic here?